README.md

```diff
@@ -1,31 +1,208 @@
+article in [JSON-LD]: 
+
+```json
+{
+  "@context": "http://schema.org/",
+  "@id": "http://example.com/my-article-store/articles/1234567890",
+  "@type": "Article",
+  "name": "My article"
+}
+```
+
+The API uses the [Hydra vocabulary] as its [hypermedia format][HATEOAS]. It also follows the [Libero API specification].
+
+It's written in [TypeScript], uses the [Koa framework][Koa], and various [RDF/JS libraries][RDF/JS].
+
+The app persists articles in a [PostgreSQL] database.
+
+<details>
+
+<summary>Further reading</summary>
+
+- [Libero API Specification]
+- [RDF 1.1 Primer]
+- [Hydra Core Vocabulary][Hydra vocabulary]
+- [RDF JavaScript Libraries][RDF/JS]
+  - [Data Model Specification][RDF/JS data model]
+  - [Dataset Specification][RDF/JS dataset]
+
+</details>
+
+Table of contents
+-----------------
+
+1. [Installation](#installation)
+   1. [Running an Article Store](#running-an-article-store)
+      1. [Using the Docker CLI](#using-the-docker-cli)
+      2. [Using Docker Compose](#using-docker-compose)
+   2. [Configuration](#configuration)
+2. [Development](#development)
+   1. [Running the app](#running-the-app)
+   2. [Running the tests](#running-the-tests)
+   3. [Linting](#linting)
+3. [Contributing](#contributing)
+4. [Getting help](#getting-help)
+5. [License](#license)
+
+Installation
 ------------
 
-- [Docker](https://www.docker.com/)
-- [GNU Bash](https://www.gnu.org/software/bash/)
-- [GNU Make](https://www.gnu.org/software/make/)
-- [Node.js](https://nodejs.org/) (for development)
+You can find the app on Docker Hub: [`liberoadmin/article-store`][Docker image].
 
-Running
--------
+As it is still under heavy development, there are not yet tagged releases. An image is available for every commit.
+
+### Running an Article Store
+
+#### Using the [Docker CLI]
+
+1. Start a PostgreSQL container by executing:
+
+   ```shell
+   docker run -d --name article-store-database \
+     -e "POSTGRES_DB=article-store" \
+     -e "POSTGRES_USER=user" \
+     postgres:11.5-alpine
+   ```
+
+2. Run the database creation with an ephemeral Article Store container:
+
+   ```shell
+   docker run --rm \
+     --link article-store-database \
+     -e "DATABASE_HOST=article-store-database" \
+     -e "DATABASE_NAME=article-store" \
+     -e "DATABASE_USER=user" \
+     liberoadmin/article-store:latest npm run initdb
+   ```
+
+3. Run an Article Store container and link it to the database container:
+
+   ```shell
+   docker run \
+     --link article-store-database \
+     -e "DATABASE_HOST=article-store-database" \
+     -e "DATABASE_NAME=article-store" \
+     -e "DATABASE_USER=user" \
+     -p 8080:8080 \
+     liberoadmin/article-store:latest
+   ```
+
+4. Access the Article Store entry point:
+
+   ```shell
+   curl --verbose localhost:8080
+   ```
+
+#### Using [Docker Compose]
+
+1. Create a file called `docker-compose.yml`:
```

Comment:
Don't need instruction to create the file as it exists in the repo?

Reply:
This is a little different in that it's for running it elsewhere (uses the published image). Maybe need some clearer wording around it.

Reply:
Ah, I see. Yes, I think some clearer wording would be helpful here.

Reply:
Was the comment actually related to having accidentally committed the file? 😳 (Refs https://github.com/libero/article-store/pull/261/commits/bbded6f9aaec1e4c6affd6427ecf6ce05fb26c4d.)